saltstack/pillar/custom/devel_standalone.sls

```diff
@@ -1,12 +1,12 @@
 {# Custom Settings #}
 
 {% set branch = 'devel' %} {# Use 'release' or 'devel' #}
-{% set version = '3.19' %}
+{% set version = '3.20' %}
 {% set environment = 'dev' %} {# Use 'dev' or 'prod' #}
 {% set r_download = 'https://stat.ethz.ch/R/daily/R-devel.tar.gz' %}
```

Comment:
This should be the same as release: `https://cran.r-project.org/src/base/R-4/R-4.4.0.tar.gz`.